get-started.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Getting Started
+summary: Quick list of instructions and hints to get started with NServiceBus and the Particular Service Platform
+reviewed: 2016-12-05
+---
+
+There are a variety of starting points for NServiceBus and the platform. The path taken will depend on an individuals specific experience and preferences for learning.
+
+
+## High Level Content
+
+There are many articles that explain concepts and technologies used in the platform. Some good articles when getting started are:
```

Comment:
I'd combine these sentences as 

These articles are recommended to gain an understanding of the key concepts and technologies used within the platform.

How about 

S